spec.emu

```diff
@@ -33,6 +33,8 @@ copyright: false
             1. If _completion_ is an abrupt completion, then
               1. Return ? IteratorClose(_iteratorRecord_, _completion_).
       1. Return ReturnCompletion(*undefined*).
-    1. Return CreateIteratorFromClosure(_closure_, *"Iterator Helper"*, %IteratorHelperPrototype%, « »).
+    1. Let _gen_ be CreateIteratorFromClosure(_closure_, *"Iterator Helper"*, %IteratorHelperPrototype%, « »).
```

Comment:
```suggestion
    1. Let _gen_ be CreateIteratorFromClosure(_closure_, *"Iterator Helper"*, %IteratorHelperPrototype%, « [[UnderlyingIterators]] »).
```

Reply:
This 100% happened because my screen wasn't wide enough to see the parameter.

<img width="896" alt="image" src="https://github.com/user-attachments/assets/0262c9db-032b-401e-a5c3-880b83d0c9b0">